account_journal_report/tests/test_account_journal_report.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openerp.tests.common import TransactionCase
+
+
+class TestAccountJournalReport(TransactionCase):
+    """
+        Tests for Account Journal Report.
+    """
+    def setUp(self):
+        super(TestAccountJournalReport, self).setUp()
+
+        self.res_users_model = self.env['res.users']
+        self.company = self.env.ref('base.main_company')
+        self.g_account_user = self.env.ref('account.group_account_user')
+
+        self.user = self._create_user('user_1', [self.g_account_user],
```

Comment:
For assuring the test, you must provide account data. Please check my tests for v8: https://github.com/OCA/l10n-spain/blob/8.0/l10n_es_account_financial_report/tests/test_journal_entries_report.py, that adds all the needed records.